keyboards/kbdfans/readme.md

```diff
@@ -0,0 +1,8 @@
+# KBDFans
+
+KBDfans is a keyboard retailer located in China and is run by Wei. 
+
+**Discord:** https://discord.gg/HMZKDhn  
+**Facebook:** https://www.facebook.com/KBDfanskeyboard/  
+**Instagram:** https://www.instagram.com/kbdfans.cn/  
+**Website:** https://kbdfans.cn/  
```

Comment:
The website should probably be listed first.

Reply:
I thought about that then I started thinking too hard about what should go next, and after that. So I figured I'd just put it in alphabetically.

Reply:
Hehe, I can relate. If it helps, I'd order them like this: Website, Discord, Facebook, Instagram. In other words, whichever is “most relevant” (the website) and wherever you're most likely to get an answer (Discord) first.
You could also add links to their [AliExpress](https://kbdfans.aliexpress.com/), [eBay](https://www.ebay.com/str/KBDfans/) and [Taobao](https://yikewaishe.taobao.com/) stores (in that order). So, all in all:
```markdown
**Website:** https://kbdfans.cn/  
**Discord:** https://discord.gg/HMZKDhn  
**Facebook:** https://www.facebook.com/KBDfanskeyboard/  
**Instagram:** https://www.instagram.com/kbdfans.cn/  
**AliExpress:** https://kbdfans.aliexpress.com/  
**eBay:** https://www.ebay.com/str/KBDfans/  
**Taobao:** https://yikewaishe.taobao.com/  
```

Reply:
Yeah... _beta_